setup.cfg

```diff
@@ -30,7 +30,7 @@ install_requires =
     appdirs
     click
     click-didyoumean
-    dandischema
+    dandischema ~= 0.1.0
```

Comment:
This is equivalent to `>= 0.1.0, <0.2.0`.  Is that what you meant, or did you mean `~= 0.1` (which is `>= 0.1, <1.0`)?

Reply:
0.2.0 broke it.  so I guess we will need to resort for now for this level.

Reply:
but I filed https://github.com/dandi/dandischema/issues/22 which could be assisted with by https://github.com/dandi/dandischema/issues/21 ;-)